packages/core/src/App/Containers/Modals/mt5-notification/mt5-notification.jsx

```diff
@@ -0,0 +1,88 @@
+        }
+    };
+
+    return (
+        <React.Suspense fallback={<UILoader />}>
+            <DesktopWrapper>
+                <Modal
+                    disableApp={disableApp}
+                    enableApp={enableApp}
+                    has_close_icon={false}
+                    is_open={is_mt5_notificaiton_modal_visible}
+                    title={localize('Trouble accessing Deriv MT5 on your mobile?')}
+                    toggleModal={onclickFunc}
+                    className='mt5-notification-modal'
+                    height='455px'
+                    width='510px'
+                >
+                    <Modal.Body has_separator className='mt5-notification-modal-body'>
+                        <MT5NotificationDescription />
+                    </Modal.Body>
+                    <Modal.Footer has_separator>
+                        <Button
+                            primary
+                            onClick={() => {
+                                setMT5NotificationModal(false);
+                            }}
+                        >
+                            {localize('OK')}
+                        </Button>
+                    </Modal.Footer>
+                </Modal>
+            </DesktopWrapper>
+            <MobileWrapper>
+                <MobileDialog
+                    portal_element_id='modal_root'
+                    wrapper_classname='mt5-notification-modal-mobile'
+                    title={localize('Trouble accessing Deriv MT5 on your mobile?')}
+                    visible={is_mt5_notificaiton_modal_visible}
+                    onClose={() => {
+                        setMT5NotificationModal(false);
+                    }}
+                    has_full_height
+                    has_close_icon={false}
+                    footer={
+                        <div className='mt5-notification-modal-mobile--footer'>
+                            <Button
+                                className='mt5-notification-modal-mobile--button'
+                                primary
+                                onClick={() => setMT5NotificationModal(false)}
+                            >
+                                {localize('OK')}
+                            </Button>
+                        </div>
+                    }
+                >
+                    <MT5NotificationDescription />
+                </MobileDialog>
+            </MobileWrapper>
+        </React.Suspense>
+    );
+};
+
+MT5Notification.propTypes = {
+    disableApp: PropTypes.func,
+    enableApp: PropTypes.func,
+    setMT5NotificationModal: PropTypes.func,
+    is_mt5_notificaiton_modal_visible: PropTypes.bool,
+};
+
+export default connect(({ traders_hub, ui }) => ({
+    setMT5NotificationModal: traders_hub.setMT5NotificationModal,
+    is_mt5_notificaiton_modal_visible: traders_hub.is_mt5_notificaiton_modal_visible,
+    disableApp: ui.disableApp,
+    enableApp: ui.enableApp,
+}))(MT5Notification);
```

Comment:
@hamza-deriv Is there any reason for using `connect` and `jsx` for the new component? 🤔

Reply:
It is in the core and we dont have access to the stores.
Thats why

Reply:
I don't think that's true 👀 Please avoid using `connect` and use `TypeScript` for newly added files 🙏🏻 

<img width="461" alt="image" src="https://github.com/binary-com/deriv-app/assets/72082844/a1a0e71e-488a-4d66-ac3b-022e8dcbc7b7">

Reply:
Done